### PR TITLE
Add output of `lodctr -q` to flare, to make sure counters are enabled

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?

on windows, add output of `lodctr -q` to flare.  Additional debugging to troubleshoot
perf counter problems.

### Motivation

get add'l information without pestering customer

### CODE REVIEW

File: pkg/flare/archive_win.go [hunk 1]
```diff
@@ -100,6 +100,28 @@ func zipTypeperfData(tempDir, hostname string) error {
 	}
 	return nil
 }
+func zipLodctrOutput(tempDir, hostname string) error {
+	cmd := exec.Command("lodctr", "/q")
```
Comment (hush-hush) [inline pkg/flare/archive_win.go:104]: Should we put a timeout on this ? Or are we sure it will never block ?

File: pkg/flare/archive_win.go [hunk 2]
```diff
@@ -100,6 +100,28 @@ func zipTypeperfData(tempDir, hostname string) error {
...
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		log.Warnf("Error running lodctr command %v", err)
```
Comment (hush-hush) [inline pkg/flare/archive_win.go:109]: Should we return an error here ? Seems like we're going to create a empty file later. Or do we want to create the `lodctr.txt` file with the error ?

File: pkg/flare/archive_win.go (hunk 1 above)
Comment (derekwbrown) [reply pkg/flare/archive_win.go]: I added a timeout on this and the other function.

File: pkg/flare/archive_win.go (hunk 2 above)
Comment (derekwbrown) [reply pkg/flare/archive_win.go:120]: I had to re-debug this to remember why I did it this way.  The `lodctr` command seems to return an error code even on success.  So we log the return code and keep going regardless.

**Review by hush-hush — APPROVED** [review]
Added a small detail but it look ready to merge beside that.

**Review by hush-hush — APPROVED** [review]

**Pull request merged**